src/Dumbo.php

```diff
@@ -300,6 +300,60 @@ public function onError(callable $handler): void
         $this->errorHandler = $handler;
     }
 
+    /**
+     * Serve static files from a directory
+     *
+     * @param string $urlPath The URL path to serve the static files from
+     * @param string $directory The directory containing the static files
+     */
+    public function staticFiles(string $urlPath, string $directory): void
```

Comment:
```suggestion
    public function files(string $urlPath, string $directory): void
```

Maybe this is better just being called `files`?